templates/NOTES.txt

```diff
@@ -7,9 +7,29 @@ Warning: `namespaceWhitelist` has been removed. Use multiple workspace providers
 {{- end }}
 
 {{- if .Values.dashboard }}
-Warning: The `dashboard` service will be removed in v1.21 (July 21st, 2021). You can safely delete the `dashboard` property.
+Warning: The `dashboard` service has been deprecated.
 {{- if eq .Values.ingress.useDefault false }}
-
-Breaking: All `dashboard` traffic should be routed to `cemanager` before v1.21.
+  All `dashboard` traffic can be routed to `cemanager`.
 {{- end }}
+  Dashboard is now served from `cemanager`. Delete `dashboard` in values.yaml.
+{{ end }}
+
+{{- if .Values.envproxy }}
+Warning: The `envproxy` service has been deprecated.
+{{- if eq .Values.ingress.useDefault false }}
+  Coder now supports end-to-end encryption. To migrate, expose port 5349 to `cemanager`.
+{{- else }}
+  Envproxy is now served from `cemanager`. Delete `envproxy` in values.yaml.
+{{- end}}
+{{ end }}
+
+{{- if .Values.podSecurityPolicyName }}
+Warning: `podSecurityPolicyName` is deprecated. It will be removed in v1.21 (July 21st, 2021).
```

Comment:
In general, deprecations should come with a suggestion for what to do to migrate, we shouldn't remove things unless there is an alternative?

```suggestion
Warning: `podSecurityPolicyName` is deprecated since Coder v1.20 and will be removed in v1.21.
```